### ISSUE TITLE
Running Mumble nukes all sound in Windows

### ISSUE THREAD

**Issue by sswires** [mumble-voip/mumble#1875]
Hi,

Recently I've been having trouble with Mumble running on Windows 10. When the Mumble client is running no sound is audible from Windows, including Mumble itself. The only exception to this is when running the Audio Wizard I can hear audio from the microphone and the speech sample. As soon as the wizard is complete audio is gone again.

Sometimes it's possible to get around this by disabling and re-enabling the audio device but this doesn't always work. I'm using a USB audio DAC (FiiO USB DAC-E17) which set as the default output and communications device.

**Comment by mkrautz** [comment]
It would be interesting to see your log. It's at %APPDATA%\Mumble\Console.txt.

You can start Mumble, try to reproduce the issue, and close it.  Then open the file, and cut out the relevant session and upload it as a Gist or to a pastebin.

Thanks!

**Comment by sswires** [comment]
Hi,

Thanks for the reply. I managed to get it working with an odd workaround. I disabled the output device in Windows as Mumble was running and set something else as the default communications device. Once I re-enabled my main output device audio started working as soon as I set it to the default communications device.

Here's the log from the most recent session (which includes doing the above workaround): https://gist.github.com/sswires/8df41b5e10230e1ef161
